modules/javafx.graphics/src/test/java/test/javafx/css/StylesheetTest.java

```diff
@@ -646,4 +649,140 @@ public void testRT_37301() {
         }
     }
 
+    private byte[] convertCssTextToBinary(String cssText) throws IOException {
+        File inputFile = null, outputFile = null;
+        byte[] stylesheetData;
+
+        try {
+            inputFile = File.createTempFile("convertCssTextToBinary", ".css");
+            outputFile = File.createTempFile("convertCssTextToBinary", ".bss");
```

Comment:
Instead of writing a pair of temp files, maybe you can call `CssParser.parse` to create a `Stylesheet` and then `Stylesheet.writeBinary` to write to an in memory `DataOutputStream`? Or alternatively, create the `.bss file` and add it as a resource under `src/test/resources/...`?